main.go

```diff
@@ -134,7 +136,9 @@ func main() {
 	const influxTimeout = 3 * time.Second
 	authString := ""
 	if config.InfluxUser != "" || config.InfluxPass != "" {
-		authString = fmt.Sprintf("%s:%s", config.InfluxUser, config.InfluxPass)
+		authString = fmt.Sprintf("%s:%s", config.InfluxUser, config.InfluxPass) 
+	} else if config.InfluxToken != "" {
```

Comment:
It might be nice to add a warning if you try to supply a token _and_ password, but overall 👍🏻